main.go

```diff
@@ -374,7 +376,11 @@ func hasLicense(b []byte) bool {
 	if len(b) < 1000 {
 		n = len(b)
 	}
-	return bytes.Contains(bytes.ToLower(b[:n]), []byte("copyright")) ||
+	return bytes.Contains(bytes.ToLower(b[:n]), []byte("# copyright")) ||
```

Comment:
This change doesn't seem necessary: "# copyright" also contains the string "copyright".